examples/tv-casting-app/linux/main.cpp

```diff
@@ -166,6 +194,7 @@ void PrepareForCommissioning(const Dnssd::DiscoveredNodeData * selectedCommissio
  * Accepts user input of selected commissioner and calls PrepareForCommissioning with
  * the selected commissioner
  */
+/*
```

Comment:
Commented out code should generally be just removed (git history would preseve it if really needed)

Reply:
Removed